doc/howtos/booting-and-installation.rst

```diff
@@ -27,19 +27,79 @@ additional build and debugging tools that wouldn't typically be included in a pr
 dev image also will auto-login as ``root`` at the console, something that normally would not be available
 in a production device image but is quite useful during development.
 
-Using dd to Create Bootable Media
-=================================
+
+Using bmaptool to Create Bootable Media
+=======================================
 
 Once you have the :file:`.dsk.xz` Ostro OS image you need to get it
 onto your hardware platform, typically by using removable media such as a
-USB thumb drive or SD card.  The usual way to do this is with the :command:`dd` command.
+USB thumb drive or SD card.
+
+The recommended way to do this is with the :command:`bmaptool` command from `bmap-tools`_.
+A copy of this utility is available in the :file:`deploy/tools` folder after a Yocto Project build
+for your image is finished, or you can install the bmap-tools package with your development system's
+package manager.  For example, on Ubuntu/Debian systems use ::
+
+      $ sudo apt-get install bmap-tools
+
+The ``bmaptool`` program automatically handles copying either compressed and uncompressed images to
+your removable media.  It also also uses a generated ``image.bmap`` file containing a checksum for
+itself and for all mapped data regions in the image file, making it possible to verify data integrity
+of the downloaded image. Be sure to download this ``.bmap`` file along with the image for your device.
```

Comment:
That's the case, because bmap indeed not present on download site.
In case of such cases it's always possible to use '--nobmap' flag.
Might be way to improve fallback mechanism in future.